babel.config.js

```diff
@@ -1,82 +0,0 @@
-module.exports = function(api) {
-  var validEnv = ['development', 'test', 'production']
-  var currentEnv = api.env()
-  var isDevelopmentEnv = api.env('development')
-  var isProductionEnv = api.env('production')
-  var isTestEnv = api.env('test')
-
-  if (!validEnv.includes(currentEnv)) {
-    throw new Error(
-      'Please specify a valid `NODE_ENV` or ' +
-        '`BABEL_ENV` environment variables. Valid values are "development", ' +
-        '"test", and "production". Instead, received: ' +
-        JSON.stringify(currentEnv) +
-        '.'
-    )
-  }
-
-  return {
-    presets: [
```

Comment:
Do we definitely not need/want any of these presets and plugins in the new Babel config?

Reply:
I'll see if we can add these in

Reply:
I've had a look through them, and I don't believe we need them - lots of them are included with `@babel/preset-env` anyway.